taxcalc/dropq/dropq_utils.py

```diff
@@ -91,6 +91,7 @@ def dropq_calculate(year_n, start_year,
         recs1p = Records(data=taxrec_df.copy(deep=True),
                          gfactors=growfactors_pre)
         # add one dollar to total wages and salaries of each filing unit
+        # to determine which records are taxpayers
```

Comment:
@Amy-Xu Thanks for reviewing this.  What about this:
```
# add one dollar to the income of each filing unit to determine
# which filing units undergo a resulting change in tax liability
```
We can replace "filing unit" with "taxpayer" or "tax unit" or some other term.